service/pom.xml

```diff
@@ -16,7 +16,10 @@
 
 	<properties>
 		<prometheus.version>0.16.0</prometheus.version>
-		<rocksdb.version>7.6.0</rocksdb.version>
+		<rocksdb.version>9.6.1</rocksdb.version>
```

Comment:
this probably means that an existing frontier won't work with the new version

Reply:
I think it should be ok
According to RocksDB team, it is fully backward compatible
https://github.com/facebook/rocksdb/wiki/RocksDB-Compatibility-Between-Different-Releases